src/components/common/Extensions/UserHub/partials/BalanceTab/partials/StreamsInfoRow/hooks.ts

```diff
@@ -73,20 +98,56 @@ export const useStreamingPaymentsTotalFunds = () => {
         totalClaimed,
         isAtLeastOnePaymentActive,
         ratePerSecond: ratePerSecondValue,
+        itemsCountedToAverage,
       } = await calculateTotalsFromStreams({
         streamingPayments: items,
         currentTimestamp: Math.floor(blockTime ?? Date.now() / 1000),
         currency,
         colony,
       });
 
+      const averageStreamingPerMonth = calculateAverageStreamingPayment(
+        new Date(blockTime ?? Date.now() / 1000),
+        itemsCountedToAverage,
+      );
+
+      setAveragePerMonth(averageStreamingPerMonth);
+
       setIsAnyPaymentActive(isAtLeastOnePaymentActive);
-      setTotalFunds({ totalAvailable, totalClaimed });
+      setTotalFunds({
+        totalAvailable,
+        totalClaimed,
+      });
       setRatePerSecond(ratePerSecondValue);
     },
     [blockTime, colony, currency],
   );
 
+  const getTotalActiveStreamingPayments = useCallback(
+    (items: StreamingPaymentItems) => {
+      const activeStreams = items.filter((item) => {
+        const { amountAvailableToClaim } = getStreamingPaymentAmountsLeft(
+          item,
+          Math.floor(blockTime ?? Date.now() / 1000),
+        );
+        return (
+          getStreamingPaymentStatus({
+            streamingPayment: item,
+            currentTimestamp: Math.floor(blockTime ?? Date.now() / 1000),
+            amountAvailableToClaim,
+          }) === StreamingPaymentStatus.Active
+        );
+      });
+
+      setActiveStreamingPayments(activeStreams.length);
+    },
+    [blockTime],
+  );
+
+  useEffect(() => {
+    getStreamingPaymentsByColony();
+  }, [getDataByRecipentAddress, nativeDomainId, getStreamingPaymentsByColony]);
```

Comment:
What's the reason for combining lazy query with useEffect over using a regular query?